BREEZE.rst

```diff
@@ -478,6 +481,16 @@ These are all available flags of ``build-docs`` command:
   :width: 100%
   :alt: Breeze build documentation
 
+.. _generating_short_form_names:
+
+Generating short form names for Providers
+-----------------------------------------
+
+Skip the ``apache-airflow-providers-`` from the usual provider full names.
+Now with the remaining part, replace every ``dash("-")`` with a ``dot(".")``.
```

Comment:
It’s weird to require the user to do this. Dots and dashes (and underscores) are equivalent in Python distribution names so the tool should be able to normalise them automatically.

Reply:
It's just explanation for parmeter names in Breeze. Historicallly we use "apache-airflow-providers-cncf-kubernetes` and `--package-filter` to choose which packages to build. But we already use "short version" of those packages when we run some breeze command and this change is purely UX change so that you can specify them  easier when you run command line. Instead of:

```
breeze build-docs --package-filter apache-airflow-providers-cncf-kubernetes --package-filter apache-airflow-providers-google --package-filter
```

you wil be able to specify:

```
breeze build-docs cncf.kubernetes google
```

This explanation is there to explain the user how to derive the short id from the long one used so far.